onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -723,7 +722,7 @@ def verify_t5_decoder_subgraph(graph: onnx.GraphProto, precision: Precision):
     #   ... (for each cross attention layer)
 
     # TODO: encoder_hidden_states is optional
-    expected_inputs = ["input_ids", "encoder_attention_mask", "encoder_hidden_states"]
+    expected_inputs = ["input_ids", "encoder_attention_mask"]
```

Comment:
If we remove "encoder_hidden_states", we will need update BeamSearch code, which checks the subgraph inputs and outputs.

Reply:
the support for decoder without encoder_hidden_states was added before. we can now use it with no change.